library/src/blas3/Tensile/gemm.h

```diff
@@ -4,6 +4,51 @@
 #include "Tensile.h"
 #include "rocblas-types.h"
 
+/*******************************************************************************
+ * Helper enumeration over different transpose combinations
+ ******************************************************************************/
+typedef enum transpose_mode_
+{
+    // First letter refers to A, second letter refers to B
+    NN,
+    NT,
+    TN,
+    TT,
+    NC,
+    CN,
+    TC,
+    CT,
+    CC
```

Comment:
We might want a trailing comma, for consistency

Reply:
understand I will correct it on third commit "update complex change".